lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -522,6 +522,7 @@ function dosomething_user_new_user($form, &$form_state) {
 
 function dosomething_user_new_user_attributes($form, &$form_state) {
   dosomething_user_set_global_attributes();
+  dosomething_northstar_update_user($form_state);
```

Comment:
since this is calling code in another module put this around it
`if (module_exists('dosomething_northstar')` so it doesn't fail if the module isn't there